automatic/peazip.install/peazip.install.nuspec

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>peazip.install</id>
+    <title>PeaZip (Install)</title>
+    <version>6.5.0</version>
```

Comment:
The version should really be set to the latest available on chocolatey,org (`6.4.1`), or to `0.0`.
Otherwise it will prevent the version `6.5.0` to be pushed (assuming that is the latest version available of the software).

I can make that change when we merge this one it though, if you prefer.